src/DracoPy.h

```
@@ -532,6 +569,42 @@ namespace DracoFunctions {
       }
     }
 
+    // GENERIC ATTRIBUTES
+    for (size_t j = 0; j < unique_ids.size(); ++j) {
+      draco::DataType dtype = static_cast<draco::DataType>(attr_data_types[j]);
+      int num_components = attr_num_components[j];
+      if (dtype != draco::DT_FLOAT32 && dtype != draco::DT_UINT8 && dtype != draco::DT_UINT16 && dtype != draco::DT_UINT32) {
+        // Unsupported data type, skip
```

Should this silently fail? Maybe it should be a flag `ignore_unsupported_attributes`? Otherwise there could be silent data loss.